File: src/test/java/org/sonar/plugins/xquery/TestUtils.java
/*
 * © 2013 by Intellectual Reserve, Inc. All rights reserved.
 */

package org.sonar.plugins.xquery;

import java.util.ArrayList;
import java.util.List;

public class TestUtils {
    
    public static List<String> list(String... strings) {
        List<String> list = new ArrayList<String>();
        for (String string : strings) {
            list.add(string);
        }
        return list;        
    }
}

File: src/test/java/org/sonar/plugins/xquery/TestUtils.kt
/*
 * © 2014 by Intellectual Reserve, Inc. All rights reserved.
 */

package org.sonar.plugins.xquery

import java.util.ArrayList

object TestUtils {

    fun list(vararg strings: String): List<String> {
        val list = ArrayList<String>()
        for (string in strings) {
            list.add(string)
        }
        return list
    }
}
